sql-schema/161.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `librenms`.`eventlog` ADD COLUMN `severity` INT(1) NULL DEFAULT 0 AFTER `reference`;
```

Comment:
You can't specify the database here, not everyone uses librenms.

I also still think default should be 1:

<pre>`ALTER TABLE `eventlog` ADD COLUMN `severity` INT(1) NULL DEFAULT 1 AFTER `reference`;</pre>